fairlearn/metrics/_balanced_root_mean_squared_error.py

```diff
@@ -11,17 +11,20 @@
 
 
 def balanced_root_mean_squared_error(y_true, y_pred, sample_weight=None):
-    """
+    r"""The mean of the root mean squared error (RMSE) for the positive and negative cases.
+
     Used for binary logistic regression, this computes the error as
-    [RMSE(Y=0) + RMSE(Y=1)]/2
-    The classes are constrained to be {0, 1}. The `y_true` values must
-    always be one of these, while `y_pred` can be a continuous probability
+
+    .. math::
+       \frac{RMSE(Y=0) + RMSE(Y=1)}{2}
```

Comment:
if this is actual LaTeX then it should probably have \text{} around RMSE or it will look like the product of four single letter variables.

Reply:
Good point. Fixed